pkgs/development/python-modules/scipy/default.nix

```diff
@@ -1,34 +1,41 @@
 {
   lib,
   stdenv,
-  fetchFromGitHub,
-  fetchpatch,
   fetchurl,
   writeText,
-  xcbuild,
   python,
   buildPythonPackage,
+  fetchFromGitHub,
+  fetchpatch,
+
+  # build-system
   cython,
   gfortran,
   meson-python,
   nukeReferences,
-  pkg-config,
   pythran,
-  wheel,
+  pkg-config,
```

Comment:
Also this was moved without a real reason.

Reply:
`wheel` was removed as it was useless.
`pkgs-config` was moved here because it does appear between `pythran` and `setuptools` in `build-system`.

Reply:
> `wheel` was removed as it was useless.

OK That's great :).

> `pkg-config` was moved here because it does appear between `pythran` and `setuptools` in `build-system`.

Hmm OK I see. I can live with that too..